src/test/java/donnafin/logic/commands/AppendCommandTest.java

```diff
@@ -121,4 +133,44 @@ public void appendNewAsset_changesAssetList() throws CommandException {
 
         assertNotEquals(GEORGE, personAdapter.getSubject());
     }
+
+    @Test
+    public void appendNewAsset_similarAssetName() throws CommandException {
+        AppendCommand testCommand = new AppendCommand(personAdapter, testAsset);
+        AppendCommand testCommand2 = new AppendCommand(personAdapter, similarAsset);
+        testCommand.execute(model);
+        assertCommandSuccess(testCommand2, model, "New asset added. "
```

Comment:
Could you switch out the magic literal for a reference to the possible duplicate asset String constant.

Reply:
Here and other places in the test files

Reply:
alright on it!